packages/common/src/ui-common/environment/environment.ts

```diff
@@ -68,6 +73,8 @@ export enum DependencyName {
     Logger = "logger",
     Localizer = "localizer",
     HttpClient = "httpClient",
+    StorageAccountService = "storageAccount",
```

Comment:
This should move up to the service package rather than be part of the base environment in common, since services aren't a concept in common.